Structural/Decorator/Tests/DecoratorTest.php

```diff
@@ -14,20 +14,21 @@ class DecoratorTest extends TestCase
 
     protected function setUp()
     {
-        $this->service = new Decorator\Webservice('foobar');
+        $this->service = new Decorator\Webservice('foo/bar');
     }
 
```

Comment:
You could add a test for Decorator\Webservice, so before testing decorators, you are sure that Webservice does its job.